cmd/dp-dataset-exporter/main.go

```diff
@@ -32,11 +32,18 @@ import (
 
 var (
 	// BuildTime represents the time in which the service was built
-	BuildTime string
+	//BuildTime string
 	// GitCommit represents the commit (SHA-1) hash of the service that is running
-	GitCommit string
+	//GitCommit string
 	// Version represents the version of the service that is running
-	Version string
+	//Version string
+
+	/* NOTE: replace the above with the below to run code with for example vscode debugger.*/
```

Comment:
I think we should uncomment `BuildTime`, `GitCommit` and `Version` above, and uncommet below?

Reply:
ah, yes well spotted. Did no mean to book that in. now fixed